src/components/backoffice/parametres/ParametresLayout/UserInformationCard/LinkedUserInformationCard.tsx

```diff
@@ -28,7 +28,7 @@ export const LinkedUserInformationCard = ({
 
   const assignUser = useCallback((userToAssign) => {
     if (isRoleIncluded(COACH_USER_ROLES, userToAssign.role)) {
-      const candidat: UserWithUserCandidate | UserWithUserCandidate[] =
+      const candidat: UserWithUserCandidate[] | null =
```

Comment:
`candidat` est forcément un tableau ?

Reply:
Oui getRelateduser renvoie toujours un tableau

Reply:
Et d'ailleurs en dessous c'est traité avec un map

Reply:
Il renvoie un tableau que dans le cas ou on est un coach non ? Il renvoie juste l'objet candidat si l'utilisateur est un candidat (propriété `candidat` de User)

Reply:
Non les 2

`if (member) {
    if (member.candidat && member.candidat.coach) {
      return [member.candidat.coach];
    }
    if (member.coaches && member.coaches.length > 0) {
      // @ts-expect-error after enable TS strict mode. Please, try to fix it
      return member.coaches.map(({ candidat }) => {
        return candidat;
      });
    }
  }`